WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -22,9 +22,6 @@
     <Style Selector="TextBox">
       <Setter Property="FontSize" Value="{DynamicResource FontSizeNormal}" />
     </Style>
-    <Style Selector="TextBox.Transparent">
-      <Setter Property="Background" Value="Transparent" />
```

Comment:
Transparent backgrounds were added because if hit-tests. For example, click or hovering was only detected if there was content in the TextBox, or the background was set to Transparent. Are you sure this can be removed?

Reply:
Yes, i relocated it to Styles.xaml, see `:is(TextBox).Transparent`